scripts/install_apt_deps.sh

```diff
@@ -1,21 +1,92 @@
 #!/bin/bash
 
+
+### Initial setup ###
+
+# Source environment variables describing the current OS distribution
 . /etc/os-release
 
+
+### Package dependencies setup ###
+
+# List of common package dependencies for the ILLIXR project
+pkg_dep_list_common="
+    build-essential
+    git
+    clang-10
+    make
+    cmake
+    unzip
+    wget
+    xvfb
+    pkg-config
+    libc++-dev libc++abi-dev
+    libsqlite3-dev
+    libeigen3-dev
+    libboost-all-dev
+    libatlas-base-dev
+    libsuitesparse-dev
+    libblas-dev
+    libtbb-dev
+    libsdl2-dev
+    libglu1-mesa-dev mesa-common-dev
+    libglew-dev glew-utils libglfw3-dev freeglut3-dev glslang-tools
+    libusb-dev libusb-1.0 libudev-dev libv4l-dev libhidapi-dev
+    libx11-xcb-dev libx11-dev libxcb-glx0-dev libxcb-randr0-dev libxrandr-dev
+    libxkbcommon-dev
+    libwayland-dev
+    libepoxy-dev
+    libdrm-dev
+    libgbm-dev
+    libvirglrenderer-dev
+    libpulse-dev
+    libgtest-dev
+    libgtk2.0-dev
+    libjpeg-dev libpng-dev libtiff-dev libvtk6-dev
+    libgtk-3-dev
+    libparmetis-dev
+    libgflags-dev libgoogle-glog-dev gfortran
+" # End list
+
+# Flatten the package dependency list string to a single line
+pkg_dep_list="$(echo ${pkg_dep_list_common} | xargs) "
+
+
+### IntelRealSense package dependencies setup ###
+
+pkg_dep_list_realsense="librealsense2-dkms librealsense2-utils"
+pkg_install_url_realsense="https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md"
+pkg_build_url_realsense="https://github.com/IntelRealSense/librealsense/blob/master/doc/installation.md"
+pkg_warn_msg_realsense="Currently, IntelRealSense does not support binary package installations for Ubuntu 20 LTS kernels, or other non-Ubuntu Linux distributions. If your project requires IntelRealSense support, please build and install the IntelRealSense SDK from source. For more information, visit '${pkg_install_url_realsense}' and '${pkg_build_url_realsense}'."
+
+# Check for distribution support of IntelRealSense
+# For supported distributions, automatically add the required install
+# package dependencies
+echo "Detected OS: '${PRETTY_NAME}'"
+if [ "${ID}" == "ubuntu" ] && [ "${VERSION_ID}" == "18.04" ]; then
+    use_realsense="yes"
+    pkg_dep_list+="${pkg_dep_list_realsense} "
+elif [ "${ID}" == "ubuntu" ] && [ "${VERSION_ID}" == "20.04" ]; then
+    use_realsense="no"
+    print_warning "${pkg_warn_msg_realsense}"
+else
+    use_realsense="no"
+    print_warning "${pkg_warn_msg_realsense}"
```

Comment:
Both branch paths are functionally equivalent, so why the distinction?

Reply:
I assumed this would change for 20.04 in the near future (I believe the Intel RealSense SDK will be getting an update based on the external repo issues referenced in #176, so I left that branching path in. Remove still?

Reply:
Also, we may have the option of doing the "backend" build process specifically only for Ubuntu 20.04 in the branch,

Reply:
If they add 20.04 support, the `if` condition changes, not the `elif`. But I see your point about manually building on 20.04. How hard is the 20.04 build? Is it robust enough that we can safely put it in a script? If yes, leave the `elif` in. If no, remove it -- best not to waste time doing error-prone builds.